### ISSUE TITLE
Appendix numbering

### ISSUE THREAD

**Issue by jomigo96** [ekspek/ist-thesis#10]
![Capture](https://user-images.githubusercontent.com/37877964/58904437-6cb38e80-86ff-11e9-91d0-67f9ffd56976.PNG)

**Comment by jomigo96** [comment]
Produced with:
```tex
\chapter{Appendix}
\appendix
\section{database}
```

**Comment by ekspek** [comment]
```\appendix``` comes first before any sections and chapters. The command does not typeset anything, but initiates the separate numbering style (among other things), so only chapters, sections and lower that come after it are affected. ```\part``` is unaffected though.

i.e. if I modify your example,
![git1](https://user-images.githubusercontent.com/37878283/59031360-bddc9300-8863-11e9-92f6-168a75392f38.png)

I used this code.
````tex
\appendix
\chapter{First Appendix}
\section{Database}
````

Personally, I created a separate part for this stuff, with all the appendices categorized as chapters. I can include the bibliography because it usually is used as an unnumbered chapter, and so doesn't get mixed with the appendix.
![git2](https://user-images.githubusercontent.com/37878283/59031595-6985e300-8864-11e9-9612-f068cf9a7dbb.png)
````tex
\part{Bibliography and Appendices}

%<bibliography code>

\appendix
\chapter{Database}
````
